File: tests/xor.cs
using System;

class Xor {
    static void Main() {
        byte[] buf = new byte[16] {0x0b,0x1a,0x29,0x38,0x4f,0x5e,
        0x6d,0x7c,0x83,0x92,0xa1,0xb0,0xc7,0xd6,0xe5,0xf4};

        for (int i = 0; i < buf.Length; i++) {
            buf[i] = (byte)(buf[i] ^ 11);
        }

        for (int i = 0; i < buf.Length; i++) {
            Console.Write(String.Format("0x{0:X2},", buf[i]));
        }
        Console.WriteLine();
    }
}

File: tests/xor.c
#include <stdio.h>

int main() {
    unsigned char buf[] =
    "\x0b\x1a\x29\x38\x4f\x5e\x6d\x7c\x83\x92\xa1\xb0\xc7\xd6"
    "\xe5\xf4";

    for (int i = 0; i < (sizeof buf - 1); i++) {
        buf[i] = buf[i] ^ 11;
    }

    for (int i = 0; i < (sizeof buf - 1); i++) {
        printf("\\x%02x", buf[i]);
    }
    printf("\n");

    return 0;
}